guestbook/static/js/guestbookapp/view/templates/GuestbookView.html

```diff
@@ -0,0 +1,75 @@
+<div>
+    <div>
+        <table>
+            <tr><td>Guestbook name: <td>
+                <td><input data-dojo-attach-point="guestbook_nameNode" id="guestbook"
+                           data-dojo-type="dijit/form/TextBox" value="${guestbook_name}" required="true" /></td>
+                <td><button data-dojo-attach-point="get_listNode" data-dojo-type="dijit/form/Button" >Get List of Greeting</button></td>
+                <td><button data-dojo-attach-point="show_dialogNode" data-dojo-type="dijit/form/Button" type="button" onClick="myDialog.show();">
```

Comment:
không bắt sự kiên trong template, handle toàn bộ sự kiện trong View
this.own( )